web/src/app/alerts/components/AlertDetailLogs.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import p from 'prop-types'
+import Divider from '@material-ui/core/Divider'
+import List from '@material-ui/core/List'
+import ListItem from '@material-ui/core/ListItem'
+import ListItemText from '@material-ui/core/ListItemText'
+import Button from '@material-ui/core/Button'
+import moment from 'moment'
+import gql from 'graphql-tag'
+import { useQuery } from '@apollo/react-hooks'
+
+const LIMIT = 149
+
+const query = gql`
+  query getAlert($id: Int!, $input: AlertRecentEventsOptions) {
+    alert(id: $id) {
+      data: recentEvents(input: $input) {
+        nodes {
+          timestamp
+          message
+        }
+        pageInfo {
+          hasNextPage
+          endCursor
+        }
+      }
+    }
+  }
+`
+export default function AlertDetailLogs(props) {
+  let logs = []
+  const { data, startPolling, stopPolling, fetchMore } = useQuery(query, {
```

Comment:
Instead of calling `startPolling` and `stopPolling` explicitly doesn't the default poll interval already handle it for us?

Reply:
Yep, it does, so we should be able to leave this out. That is, unless we'd like to stop polling if we load more results